internal/service/ipam/api_subnet_resource_test.go

```diff
@@ -903,19 +913,13 @@ resource "bloxone_ipam_subnet" "test" {
 
 func testAccSubnetSpace(space string) string {
 	config := fmt.Sprintf(`
-resource "bloxone_ipam_ip_space" "one" {
-    name = %q
-}
-resource "bloxone_ipam_ip_space" "two" {
-    name = %q
-}
 resource "bloxone_ipam_subnet" "test" {
     address = "10.0.0.0"
     cidr = 24
     space = %s.id
 }
-`, acctest.RandomNameWithPrefix("ip-space"), acctest.RandomNameWithPrefix("ip-space"), space)
-	return strings.Join([]string{testAccBaseWithIPSpace(), config}, "")
+`, space)
+	return strings.Join([]string{testAccBaseWithIPSpace(), testAccMultiSpace(acctest.RandomNameWithPrefix("ip-space"), acctest.RandomNameWithPrefix("ip-space")), config}, "")
```

Comment:
nit: you didn't have to pass the space names, and could have generated in the new function you created 
Also, you don't need testAccBaseWithIPSpace for this test

Reply:
Updated